BitWatch/HomeViewController.swift

```diff
@@ -9,30 +9,18 @@
 import UIKit
 import BitWatchKit
 
-class HomeViewController: UIViewController {
 
+class HomeViewController: UIViewController, HomeVCDelegate{
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // Do any additional setup after loading the view.
         let height = self.view.frame.size.height;
         let width = self.view.frame.size.width;
         
```

Comment:
I like to compartmentalize the whole view as it's own file. This way all the view logic and subviews don't clutter up the VC.